kafka-ui-api/src/test/java/com/provectus/kafka/ui/util/AccessControlServiceMock.java

```diff
@@ -0,0 +1,25 @@
+package com.provectus.kafka.ui.util;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+import com.provectus.kafka.ui.service.rbac.AccessControlService;
+import org.mockito.Mockito;
+import reactor.core.publisher.Mono;
+
+public class AccessControlServiceMock {
+
+  private static AccessControlService mock = Mockito.mock(AccessControlService.class);
```

Comment:
not sure why we need static instance here

Reply:
done